views/js/pciCreator/dev/mathEntryInteraction/runtime/mathEntryInteraction.js

```diff
@@ -518,13 +526,13 @@ define([
         createToolbar: function createToolbar() {
             var self = this,
                 availableTools = {
-                    frac:   { label: '<sup>x</sup>&frasl;<sub>y</sub>',latex: '\\frac',fn: 'cmd',desc: 'Fraction' },
+                    frac:   { label: 'x/y',         latex: '\\frac',    fn: 'cmd',      desc: 'Fraction' },
                     sqrt:   { label: '&radic;',     latex: '\\sqrt',    fn: 'cmd',      desc: 'Square root' },
                     exp:    { label: 'x&#8319;',    latex: '^',         fn: 'cmd',      desc: 'Exponent' },
                     log:    { label: 'log',         latex: '\\log',     fn: 'cmd',      desc: 'Log' },
                     ln:     { label: 'ln',          latex: '\\ln',      fn: 'cmd',      desc: 'Ln' },
-                    e:      { label: '&#8494;',     latex: '\\mathrm{e}',fn: 'write',   desc: 'Euler\'s constant' },
-                    infinity: { label: '&#8734;',   latex: '\\infty',   fn: 'cmd',      desc: 'Infinity' },
+                    e:      { label: 'e',           latex: '\\mathrm{e}',fn: 'write',   desc: 'Euler\'s constant' },
```

Comment:
e is different than `&#8494` which is an estimated symbol. Is this a requested change?

Reply:
I changed it according to
https://github.com/oat-sa/extension-tao-itemqti-pci/pull/196#discussion_r423274864

Reply:
Ah ok we are using `e` for Euler's number then ok